wire-ios-transport/Source/Public/ZMUpdateEvent.swift

```diff
@@ -244,7 +244,7 @@ open class ZMUpdateEvent: NSObject {
     /// True if this event type could have two versions, encrypted and non-encrypted, during the transition phase
     open var hasEncryptedAndUnencryptedVersion: Bool {
         switch self.type {
-        case .conversationOtrMessageAdd,
+        case .conversationOtrMessageAdd, // .conversationMLSMessageAdd?
```

Comment:
I guess it makes sense but we can delete this property, it was used during a transition phase when Wire introduced encryption.